### PR TITLE
doc: remove Tor link & generalize onion getnodeaddresses RPC

### PR DESCRIPTION
**Pull request by MarnixCroes**
- remove broken link about how to properly configure tor
- generalize getnodeaddresses RPC in doc

### CODE REVIEW

**Review by jonatack — COMMENTED** [review]
Concept ACK

**Comment by MarnixCroes** [comment]
@jonatack thank you! I've applied your suggestions

File: doc/cjdns.md [hunk 1]
```diff
@@ -112,5 +112,7 @@ There are several ways to see your CJDNS address in Bitcoin Core:
 To see which CJDNS peers your node is connected to, use `bitcoin-cli -netinfo 4`
 or the `getpeerinfo` RPC (i.e. `bitcoin-cli getpeerinfo`).
 
-To see which CJDNS addresses your node knows, use the `getnodeaddresses 0 cjdns`
-RPC.
+To fetch a number of CJDNS peers that your node knows, for example 7 addresses, run
+`bitcoin-cli getnodeaddresses 7 cjdns`. You may also run
+`bitcoin-cli getnodeaddresses 0 cjdns` to return all of them. These results, as well
```
Comment (fanquake) [inline doc/cjdns.md:117]: `These results, as well as those of -addrinfo, are filtered for quality and recency.`  - can you drop this. It's not clear why we need to mention an implemention detail of `-addrinfo` here.

Comment (fanquake) [reply doc/cjdns.md]: > You can use the getnodeaddresses RPC to fetch a number of onion peers known to your node; run bitcoin-cli help getnodeaddresses for details.

If anything, this seems better than reverting to the previous text.

**Review by fanquake — APPROVED** [review]
ACK 6fce5ddc17ac9d1e07849f92088ea3f7cfcafe26

**Pull request merged**